src/server/serverSetup.js

```diff
@@ -41,15 +46,69 @@ fluid.defaults("sjrk.storyTelling.server", {
                     secrets: "@expand:sjrk.storyTelling.server.resolveJSONFile({that}.options.secureConfig.secretsConfigPath)"
                 },
                 port: "{that}.options.globalConfig.port",
+                session: {
+                    store: "@expand:sjrk.storyTelling.server.makeMemorySessionStore({that}.options.globalConfig.sessionCheckPeriod)",
+                    middlewareOptions: {
+                        secret: "{server}.options.secureConfig.secrets.session"
+                    }
+                },
+                account: "{that}.options.secureConfig.secrets.account",
```

Comment:
What is this account for? `secrets.json.example` does not provide an example of this account value.

This affects line 69-72 too which distributes this value.

Reply:
This provides a means of configuring [`fluid.express.user.utils`](https://github.com/fluid-project/fluid-express-user/blob/e826290beb4fdb416a4fedf602803ac1bcef4f37/src/js/server/utils.js#L28). Not sure how to show that in the `secrets.json.example`. Any suggestions?

Reply:
Can you point me to the doc or code that show how this `account` is used for configuring `fluid.express.user.utils`? I didn't find its usage in the given link. Just want to get an idea about its value type and the logic behind it. Thanks.

Reply:
@cindyli the link above is to the components defaults. We add fluid.express.user.utils as a [subcomponent](https://github.com/jobara/sjrk-story-telling/blob/89fe0ec255b7fef24aa12c107a7f3b198ac257df/src/server/serverSetup.js#L76-L102) to the server. These defaults can be set/overridden by passing in configuration to the secrets.json file under the `account` property. These are relayed via the lines you mentioned above. The component options for [account](https://github.com/jobara/sjrk-story-telling/blob/89fe0ec255b7fef24aa12c107a7f3b198ac257df/src/server/serverSetup.js#L55) and the [options distributions](https://github.com/jobara/sjrk-story-telling/blob/89fe0ec255b7fef24aa12c107a7f3b198ac257df/src/server/serverSetup.js#L69-L72) to send this to the instance of fluid.express.user.utils.

Reply:
I think now I understand better of what you said about "provides a means of configuring fluid.express.user.utils". A couple of suggestions:

1. Please consider to use a more meaningful variable name such as `expressUserUtilsConfig`;
2. I'm not sure if it's a good idea to use `secrets.json` to distribute component options. These options only make sense to SJRK developers while `secrets.json` gives a feeling of being configured by website admins who unlikely know about infusion and what component options are;
3. If you decided to use `secrets.json` for this config, please add documentation, for example in README where `secrets.json` is explained. (If `secrets.json` is a json5 file, comments for each value in that file would be easier to access.)

Reply:
@cindyli I've added some documentation about configuration, add some examples to the secrets.json.example file, and renamed some things. I think mostly you can just use the kettle config to configure expressUserUtils, but there are some parts that we may want to express in the secrets file, so I've illustrated those in the example.